### DISCUSSION TITLE
¿I can force load file to address?

### DISCUSSION THREAD

**Discussion by edudele** [john30/ebusd#654] [Q&A] [answered]
After time fiddling with files and the daemon.
I would like to know if it is possible to force the loading of a file in one direction. for instance:

> address 15: slave #2, scanned "MF=Vaillant;ID=BASV0;SW=0217;HW=8803"

I would like to force the loading of the 17.500.csv file since it does not take it by itself.

**Comment by john30** [comment, marked as answer]
if you think it matches with the 15.700.csv (not 17.500.csv, right?) then you could try that by cloning the ebusd-configuration repo, add a link in the 2.1.x/de/vaillant (or ...en...) folder to 15.700.csv as 15.basv.csv, point ebusd to the de (or en) folder and check the messages
